samples/subsys/usb/dfu/README.rst

```diff
@@ -86,7 +86,7 @@ showing this output to the console:
   [MCUBOOT] [INF] boot_status_source: Image 0: magic=unset, copy_done=0xff, image_ok=0xff
   [MCUBOOT] [INF] boot_status_source: Scratch: magic=unset, copy_done=0xe, image_ok=0xff
   [MCUBOOT] [INF] boot_status_source: Boot source: slot 0
-  [MCUBOOT] [INF] boot_swap_type: Swap type: test
+  [MCUBOOT] [INF] mcuboot_swap_type: Swap type: test
```

Comment:
This should not have been changed; those log messages are from mcuboot itself

Reply:
fix covered in #13552